src/components/appointments/Appointments.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  fetchAppointments,
+  selectAppointments,
+  selectAppointmentsStatus,
+  selectAppointmentsError,
+  deleteAppointment,
+} from '../../redux/appointmentsSlice';
+import Navigation from '../Navigation';
+
+function Appointments() {
```

Comment:
- [ ] It seems that all the users share the same appointments.  I tested this by creating multiple users and creating an appointment with one of them:

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/d60806cb-0779-4f4f-a156-82e7b4618e78)

![image](https://github.com/CaptainBawa/DocBook-Front-End/assets/57421823/52d79715-5bc6-4c96-ae62-9671b234334f)

Kindly, make it so each user can have their own appointments.  Tip: The backend work for this is already done and present in your backend project.

That way, your app will be more useful. 👍